clang/include/clang/Sema/Initialization.h

```diff
@@ -205,8 +209,8 @@ class alignas(8) InitializedEntity {
     /// virtual base.
     llvm::PointerIntPair<const CXXBaseSpecifier *, 1> Base;
 
-    /// When Kind == EK_ArrayElement, EK_VectorElement, or
-    /// EK_ComplexElement, the index of the array or vector element being
+    /// When Kind == EK_ArrayElement, EK_VectorElement, or EK_MatrixElement,
```

Comment:
nit: the word "or" appears twice
```suggestion
    /// When Kind == EK_ArrayElement, EK_VectorElement, EK_MatrixElement,
```